changelog.md

```diff
@@ -15,6 +15,8 @@
 - Fix formatting bug in the 'vmq-admin trace` command.
 - Handle empty modifier list correctly in `vmq_webhooks` (#339).
 - Handle client_id and mountpoint modifiers correctly in `vmq_webhooks` (#332).
+- Fix bug where an aborted connection handshake caused queued offline messages 
+  not being migrated to remote node.
```

Comment:
I'd rather say `migrated properly` as the messages are migrated to the remote node, they're just not available in the queue.